api/users.go

```diff
@@ -28,6 +28,13 @@ func addUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	editor := context.Get(r, "user").(*db.User)
+	if editor.Admin != true {
+		log.Warn(editor.Username + " doesn't permitted for user creating")
```

Comment:
should be `log.Warn(editor.Username + " doesn't permit user creation")`
Please correct this for all other instances/variations of this text as well

Reply:
Done